lib/bindings/HTTPBindings.js

```diff
@@ -276,6 +276,10 @@ function generateCommandExecution(apiKey, device, attribute) {
         }
     };
 
+    if (config.getConfig().http.timeout) {
```

Comment:
IOTAs can be configured by config.js file and by environement variables (the later overrides the former) so a new environment varible should be defined for the new parameter (suggested name IOTA_HTTP_TIMEOUT).

In addition both config.js parameter and env var should be added to the documentation at the following section: https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#configuration

Reply:
Fixed in 39779bd